ml/models/logo_detection/process.py

```diff
@@ -0,0 +1,12 @@
+import torch
+from detect import Detector  # Importing the Detector class from detect.py
```

Comment:
Same as import above
```suggestion
from .detect import Detector
```